extension/src/helpers/stellar.ts

```diff
@@ -56,19 +55,12 @@ export const getTransactionInfo = (search: string) => {
   };
 };
 
-export const getAssetFromCanonical = (canonical: string) => {
+export const getAssetFromCanonical = (canonical: string): Asset => {
   if (canonical === "native") {
     return StellarSdk.Asset.native();
   }
   if (canonical.includes(":")) {
     const [code, issuer] = canonical.split(":");
-
-    if (isSorobanIssuer(issuer)) {
```

Comment:
hm, I think you may need to rebase or something - I don't think you want to drop this

Reply:
oh actually this is intentional. While working in this file I noticed that there isn't a difference in the canonical for a Soroban asset(still can be represented as a code/issuer) so I simplified this a bit. It might be more clear to look [at this commit in isolation](https://github.com/stellar/freighter/pull/778/commits/91fae58372661ae52f852470ff5e63cbc6933363)

Apologies, maybe I should have pulled this change out on its own since it doesn't really follow the theme of the PR.

Reply:
No worries - just making sure!